README.md

```diff
@@ -32,7 +32,7 @@ Deployment packages as well as source code will be available in the **dist** dir
 ## Deployment guide
 A deployment guide is available from the Fortinet Document Library:
 
-  + [ FortiGate / FortiOS 6.2 / Deploying auto scaling on Azure](https://docs.fortinet.com/vm/azure/fortigate/6.2/azure-cookbook/6.2.0/161167/deploying-auto-scaling-on-azure)
+  + [ FortiGate / FortiOS 6.4 / Deploying auto scaling on Azure](https://docs2.fortinet.com/vm/azure/fortigate/6.4/azure-cookbook/6.4.0/161167/deploying-auto-scaling-on-azure)
```

Comment:
Please abandon this change.
Azure is not officially supported on FortiOS 6.4.

Reply:
When FortiOS 6.4 is officially supported, the 6.4 documentation link can be added. when added, the link should start with https://docs.fortinet.com, not docs2.